templates/semantic-ui/invenio_requests/legacy-record-upgrade/index.html

```diff
@@ -0,0 +1,14 @@
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
```

Comment:
very minor: Not sure if `is part of Invenio` or `InvenioRDM` or `ZenodoRDM` 😂 
Sometimes it even says `Invenio App RDM`

Reply:
Good point! I would stick to `Zenodo` as this repo belongs to `zenodo/zenodo-rdm`